### PR TITLE
Remove irc stream task from iced stream which has backpressure

### PR DESCRIPTION
**Pull request by tarkah**

### CODE REVIEW

File: data/src/stream.rs [hunk 1]
```diff
@@ -75,7 +75,12 @@ pub fn run(
 ) -> impl futures::Stream<Item = Update> {
     let (sender, receiver) = mpsc::unbounded();
 
-    let runner = stream::once(_run(server, proxy, sender)).map(|_| unreachable!());
+    let runner = stream::once(async {
+        // Spawn to unblock backend from iced stream which
+        // has backpressure
+        tokio::spawn(_run(server, proxy, sender));
+        future::pending().await
```
Comment (hecrj) [inline data/src/stream.rs:82]: I believe we can simply `await` the `spawn` result here.

**Review by hecrj — CHANGES_REQUESTED** [review]

File: data/src/stream.rs (hunk 1 above)
Comment (tarkah) [reply data/src/stream.rs:82]: Won't that tie that future back to the stream?

Comment (tarkah) [reply data/src/stream.rs:82]: We need it to stay detached

Comment (hecrj) [reply data/src/stream.rs:82]: No, the handle returned is a future that resolves once the spawned task finishes. It is not the task itself.

Comment (tarkah) [reply data/src/stream.rs]: Yup makes sense, I just read the docs further. Thanks!

**Pull request merged**